src/pages/RoadObstructions/components/DetailFeature/DetailFeature.test.tsx

```diff
@@ -48,11 +48,15 @@ describe('DetailFeature', () => {
       await user.click(roadSections[0])
     })
 
-    act(() => {
-      expect(window.scrollTo).toHaveBeenCalled()
-    })
+    try {
```

Comment:
(Multiple instances) Why do we need a try-catch in a test? We expect something right. Either an error or not an error. Seems superfluous to catch errors in tests. The testing library should have a function to test if an error occurred. Errors should be logged without the console.error

Reply:
The change in how we handle the envs caused tests to fail. I've been debugging them for way too long and ended up with this solution.

The tests are all a bit shaky anyway with all the DOM query methods and such that shouldn't be used. The end-to-end tests are in need of a complete overhaul.

Reply:
The try-catch statements have been removed. Found a solution for env var handling.